models/image_segmentation_efficientsam/demo.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import numpy as np
+import cv2 as cv
+from efficientSAM import EfficientSam
+
+# Check OpenCV version
+assert cv.__version__ >= "4.9.0", \
+       "Please install latest opencv-python to try this demo: python3 -m pip install --upgrade opencv-python"
+
+parser = argparse.ArgumentParser(description='EfficientSAM Demo')
+parser.add_argument('--input', '-i', type=str,
+                    help='Set input path to a certain image, omit if using camera.')
+parser.add_argument('--model', '-m', type=str, default='image_segmentation_efficientsam_ti_2024may.onnx',
+                    help='Set model path, defaults to image_segmentation_efficientsam_ti_2024may.onnx.')
+parser.add_argument('--save', '-s', action='store_true',
+                    help='Specify to save a file with results. Invalid in case of camera input.')
+args = parser.parse_args()
```

Comment:
Should add backend and target option. https://github.com/opencv/opencv_zoo/blob/ec5b2c8af6b3505b28b7721f7d18528ac22afa2b/models/face_detection_yunet/demo.py#L32-L39